### ISSUE TITLE
> Thought about that but low key... No clue how to add hogyoku in the mod lmao.

### ISSUE THREAD

**Issue by Beosti** [Beosti/souls-awakening#14]
> Thought about that but low key... No clue how to add hogyoku in the mod lmao.

### The soul society has fallen, trillions must be sacrificed to the hogyoku.
Idk like, add a soul system for the hogyoku where if you sacrifice enough beings with reiryoku to it (have to make a hogyoku "shell" or something it goes from a hogyoku shell to an incomplete hogyoku, which gives you a free no risk method way of hollowfication for shinigami and arrancarfication for menos grande below the level of Vasto Lorde. 

If you sacrifice like somewhere between 1000 and 10000, maybe even 100000 (different entities would give a different amount of souls, with vasto lorde and such giving the most which is be canon if vasto lorde are evolved adjuchas) it becomes a complete hogyoku which lets you start merging with it, granting different buffs and forms for each race as they take damage and keep getting closer and closer to death; with the final form being a normal looking whatever which only transforms when you enter your release state (Vollstanding for Quincies, Res/Segunda for arrancars and bankai for shinigami)

The hogyoku merger should also require an amount of time spent with the hogyoku bonded to you before you can start the merging.
Would be funny if you added the muken clothes to limit the spiritual pressure of a hogyoku transformed individual.

_Originally posted by @DanielWaeraasUran in https://github.com/Beosti/souls-awakening/issues/3#issuecomment-2046562754_
            
# Hokyoku
##Explanation
tl;dr hogyoku is a "wish granting" device, that could give a bit of everything
## Uses of it
I'd like to give it multiple uses to it when having it in the inventory and "consuming it"
### Resurection
I'd like if the hogyoku has gathered enough souls and you have it in your inventory it can grant you a sort of totem of undying effect. Saving you at the brink of death but getting consumed
### Granting power to others
It'd be cool to make it so if you're a player with others that are still human, giving them the hogyoku and "consuming it" will give them very special specific powers like Chad or Urahime.
### Next level
Consuming the hogyoku as a shinigami, arrancar (or even a quincy) will make you get the next level this can be tied up to a lot of other stuff in the power system. Like this being **the** or **one of the** reasons you can get segunda etappa for the arrancar or that you can skip a stage and immediately go to arrancar (this will have downsides though) and for shinigami just give you some extra b*llshit.
## Getting it
Craft a "empty husk", while in the inventory kill a ton of stuff. Depending on the entity it has a soul value and it gets added to the hogyoku the different effects of what the hogyoku can grant are depending on the amount of souls
## Note
I'd very much like it so the hogyoku is an _option_ it doesn't **have** to be used for anything. Everything can be gotten naturally but this is a cheat way sort of.

**Comment by DanielWaeraasUran** [comment]
I think that for the whole "Consuming it" part of the hogyoku you should be able to craft a "complete hogyoku" by feeding an "incomplete" hogyoku with the maximum number of souls to another hogyoku with the same conditions.
The completed Hogyoku could work with a sort of durability system that fills out as you gain souls, and after you completely fill up the hogyoku you get the ability to merge with the hogyoku (consume it) if you're "Worthy" enough (strong enough or something idk what the hogyoku wants)

**Comment by Beosti** [comment]
tl;dr Too difficult you have to assume players are 8 years old. I want the main path (just grind mostly and do missions) to be straight forward, the hokyoku is a cheat way that's already more confusing. Don't want to make it more complicated. Really spending more time developing stuff to make it easy for the player to know what is what.

**Comment by DanielWaeraasUran** [comment]
> tl;dr Too difficult you have to assume players are 8 years old. I want the main path (just grind mostly and do missions) to be straight forward, the hokyoku is a cheat way that's already more confusing. Don't want to make it more complicated. Really spending more time developing stuff to make it easy for the player to know what is what.

I keep trying to forget what happens in the discord server, but I just can't...
Very true what you said, if people don't know how to bind keys how are they supposed to know anything?
Minecraft players truly are a massive group of people who are either 8 year olds or mentally 8 (me)
